projects/example/src/app/app.component.ts

```diff
@@ -10,9 +10,12 @@ import {SmartTableConfig} from 'projects/smart-table/src/lib/smart-table/smart-t
 export class AppComponent {
   title = 'example';
 
-  moviesCustomColumns = [{
+  moviesCustomColumns: SmartTableColumnCustomType[] = [{
     name: 'rating',
-    component: AppRatingComponent
+    component: {
```

Comment:
dit is een breaking change, moet dus op die manier aangeduid worden in de changelog en de nieuwe versie moet een major worden

misschien dat het op een non-breaking manier ook kan met "component: Component|ConstructableCell"?

Reply:
@jsebrech  Dit is geen breaking change, de oude manier zou nog steeds werken, beide kan. Dit is hier aangepast om duidelijk te maken dat je ook metadata kunt doorsturen.